packages/template-block/src/TemplateBlock.tsx

```diff
@@ -211,7 +238,7 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
                                     }
                                 />
                             </div>
-                            <div className="tw-shrink-0">
+                            <div className={hasPreview ? '' : 'tw-flex tw-justify-end tw-items-start'}>
```

Comment:
use merge here as well to keep it consistent

Reply:
Fixed